volume/drivers/pwx/connection.go

```diff
@@ -68,14 +71,15 @@ func NewConnectionParamsBuilderDefaultConfig() *ConnectionParamsBuilderConfig {
 }
 
 // NewConnectionParamsBuilder constructor function to create ConnectionParamsBuilder with needed dependencies
-func NewConnectionParamsBuilder(ops core.Ops, params *ConnectionParamsBuilderConfig) (*ConnectionParamsBuilder, error) {
+func NewConnectionParamsBuilder(ops core.Ops, params *ConnectionParamsBuilderConfig, authTokenGen func() (string, error)) (*ConnectionParamsBuilder, error) {
```

Comment:
Instead of adding it as a new parameter, add it to the ConnectionParamsBuilderConfig

Reply:
I agree, fixed